caffe-grid/src/main/python/com/yahoo/ml/caffe/DisplayUtils.py

```diff
@@ -36,6 +40,26 @@ def show_df(df, nrows=10):
     html += "</table>"
     return HTML(html)
 
+def show_captions(df, nrows=10):
```

Comment:
ok will do shortly. Fixing the dataset for test right now, as the current dataframe with images leads to OOM on Travis